packs/credential_security.yml

```diff
@@ -0,0 +1,51 @@
+AnalysisType: pack
+PackID: PantherManaged.CredentialSecurity
+Description: Detect compromised security credentials
+DisplayName: "Panther Credential Security Pack"
+PackDefinition:
+  IDs:
+    # Data Models
+    - Standard.Atlassian.Audit
+    - Standard.Github.Audit
+    - Standard.Okta.SystemLog
+    - Standard.Zendesk.AuditLog
+    # Global Helpers
+    - global_filter_auth0
+    - global_filter_github
+    - panther_auth0_helpers
+    - panther_base_helpers
+    - panther_default
+    - panther_event_type_helpers
+    # Rules
+    - Auth0.MFA.Policy.Disabled
+    - Auth0.MFA.Risk.Assessment.Disabled
+    - Asana.Workspace.Password.Requirements.Simple
+    - Dropbox.User.Disabled.2FA 
+    - Duo.Admin.App.Integration.Secret.Key.Viewed
+    - Duo.Admin.Bypass.Code.Created
+    - Duo.Admin.Bypass.Code.Viewed
+    - GitHub.Secret.Scanning.Alert.Created
+    - GSuite.LeakedPassword
+    - GSuite.Workspace.PasswordEnforceStrongDisabled
+    - GSuite.Workspace.PasswordReuseEnabled
+    - Microsoft365.MFA.Disabled
+    - Okta.Global.MFA.Disabled
+    - Okta.PasswordAccess
+    - Okta.Support.Reset
+    - Okta.User.MFA.Reset.Single
+    - Okta.User.MFA.Reset.All
+    - OneLogin.PasswordAccess
+    - OneLogin.PasswordChanged
+    - OneLogin.AuthFactorRemoved
+    - OnePassword.Lut.Sensitive.Item
+    - OnePassword.Sensitive.Item
```

Comment:
`OnePassword.Sensitive.Item` is `enabled: false`

Reply:
Definitely one that requires some custom configuration, probably ought to be dropped.